packages/s/simple-statistics.json

```diff
@@ -23,8 +23,7 @@
       {
         "basePath": "dist",
         "files": [
-          "simple_statistics.min.js",
-          "simple_statistics.js"
+          "**/*"
```

Comment:
```suggestion
          "*.@(js|mjs)?(.map)"
```
Open globs are dangerous, let's restrict this to just the extensions we need.

[Glob test](https://www.digitalocean.com/community/tools/glob#?comments=true&glob=dist%2F%2A.%40%28js%7Cmjs%29%3F%28.map%29&matches=true&tests=%2F%2F%20Imported%20from%20simple-statistics%20NPM%20package&tests=LICENSE&tests=src%2Fadd_to_mean.js&tests=src%2Fapprox_equal.js&tests=src%2Fbayesian_classifier.js&tests=src%2Fbernoulli_distribution.js&tests=src%2Fbinomial_distribution.js&tests=src%2Fbisect.js&tests=src%2Fchi_squared_distribution_table.js&tests=src%2Fchi_squared_goodness_of_fit.js&tests=src%2Fchunk.js&tests=src%2Fckmeans.js&tests=src%2Fcoefficient_of_variation.js&tests=src%2Fcombinations_replacement.js&tests=src%2Fcombinations.js&tests=src%2Fcombine_means.js&tests=src%2Fcombine_variances.js&tests=src%2Fcumulative_std_normal_probability.js&tests=src%2Fepsilon.js&tests=src%2Fequal_interval_breaks.js&tests=src%2Ferror_function.js&tests=src%2Feuclidean_distance.js&tests=src%2Fextent_sorted.js&tests=src%2Fextent.js&tests=src%2Ffactorial.js&tests=src%2Fgamma.js&tests=src%2Fgammaln.js&tests=src%2Fgeometric_mean.js&tests=src%2Fharmonic_mean.js&tests=index.js&tests=src%2Finterquartile_range.js&tests=src%2Finverse_error_function.js&tests=src%2Fk_means_cluster.js&tests=src%2Fkernel_density_estimation.js&tests=src%2Flinear_regression_line.js&tests=src%2Flinear_regression.js&tests=src%2Fmake_matrix.js&tests=src%2Fmax_sorted.js&tests=src%2Fmax.js&tests=src%2Fmean_simple.js&tests=src%2Fmean.js&tests=src%2Fmedian_absolute_deviation.js&tests=src%2Fmedian_sorted.js&tests=src%2Fmedian.js&tests=src%2Fmin_sorted.js&tests=src%2Fmin.js&tests=src%2Fmode_fast.js&tests=src%2Fmode_sorted.js&tests=src%2Fmode.js&tests=src%2Fnumeric_sort.js&tests=src%2Fperceptron.js&tests=src%2Fpermutation_test.js&tests=src%2Fpermutations_heap.js&tests=src%2Fpoisson_distribution.js&tests=src%2Fprobit.js&tests=src%2Fproduct.js&tests=src%2Fquantile_rank_sorted.js&tests=src%2Fquantile_rank.js&tests=src%2Fquantile_sorted.js&tests=src%2Fquantile.js&tests=src%2Fquickselect.js&tests=src%2Fr_squared.js&tests=src%2Frelative_error.js&tests=src%2Froot_mean_square.js&tests=src%2Fsample_correlation.js&tests=src%2Fsample_covariance.js&tests=src%2Fsample_kurtosis.js&tests=src%2Fsample_skewness.js&tests=src%2Fsample_standard_deviation.js&tests=src%2Fsample_variance.js&tests=src%2Fsample_with_replacement.js&tests=src%2Fsample.js&tests=src%2Fshuffle_in_place.js&tests=src%2Fshuffle.js&tests=src%2Fsign.js&tests=src%2Fsilhouette_metric.js&tests=src%2Fsilhouette.js&tests=dist%2Fsimple-statistics.js&tests=dist%2Fsimple-statistics.min.js&tests=src%2Fstandard_deviation.js&tests=src%2Fstandard_normal_table.js&tests=src%2Fsubtract_from_mean.js&tests=src%2Fsum_nth_power_deviations.js&tests=src%2Fsum_simple.js&tests=src%2Fsum.js&tests=src%2Ft_test_two_sample.js&tests=src%2Ft_test.js&tests=src%2Funique_count_sorted.js&tests=src%2Fvariance.js&tests=src%2Fz_score.js&tests=package.json&tests=dist%2Fsimple-statistics.js.map&tests=dist%2Fsimple-statistics.min.js.map&tests=dist%2Fsimple-statistics.mjs.map&tests=CHANGELOG.md&tests=README.md&tests=dist%2Fsimple-statistics.mjs&tests=src%2Fadd_to_mean.d.ts&tests=src%2Fapprox_equal.d.ts&tests=src%2Fbayesian_classifier.d.ts&tests=src%2Fbernoulli_distribution.d.ts&tests=src%2Fbinomial_distribution.d.ts&tests=src%2Fbisect.d.ts&tests=src%2Fchi_squared_distribution_table.d.ts&tests=src%2Fchi_squared_goodness_of_fit.d.ts&tests=src%2Fchunk.d.ts&tests=src%2Fckmeans.d.ts&tests=src%2Fcoefficient_of_variation.d.ts&tests=src%2Fcombinations_replacement.d.ts&tests=src%2Fcombinations.d.ts&tests=src%2Fcombine_means.d.ts&tests=src%2Fcombine_variances.d.ts&tests=src%2Fcumulative_std_normal_probability.d.ts&tests=src%2Fepsilon.d.ts&tests=src%2Fequal_interval_breaks.d.ts&tests=src%2Ferror_function.d.ts&tests=src%2Fextent.d.ts&tests=src%2Ffactorial.d.ts&tests=src%2Fgeometric_mean.d.ts&tests=src%2Fharmonic_mean.d.ts&tests=index.d.ts&tests=src%2Finterquartile_range.d.ts&tests=src%2Finverse_error_function.d.ts&tests=src%2Fk_means_cluster.d.ts&tests=src%2Fkernel_density_estimation.d.ts&tests=src%2Flinear_regression_line.d.ts&tests=src%2Flinear_regression.d.ts&tests=src%2Fmax_sorted.d.ts&tests=src%2Fmax.d.ts&tests=src%2Fmean_simple.d.ts&tests=src%2Fmean.d.ts&tests=src%2Fmedian_absolute_deviation.d.ts&tests=src%2Fmedian_sorted.d.ts&tests=src%2Fmedian.d.ts&tests=src%2Fmin_sorted.d.ts&tests=src%2Fmin.d.ts&tests=src%2Fmode_fast.d.ts&tests=src%2Fmode_sorted.d.ts&tests=src%2Fmode.d.ts&tests=src%2Fnumeric_sort.d.ts&tests=src%2Fperceptron.d.ts&tests=src%2Fpermutation_test.d.ts&tests=src%2Fpermutations_heap.d.ts&tests=src%2Fpoisson_distribution.d.ts&tests=src%2Fprobit.d.ts&tests=src%2Fproduct.d.ts&tests=src%2Fquantile_rank_sorted.d.ts&tests=src%2Fquantile_rank.d.ts&tests=src%2Fquantile_sorted.d.ts&tests=src%2Fquantile.d.ts&tests=src%2Fquickselect.d.ts&tests=src%2Fr_squared.d.ts&tests=src%2Frelative_error.d.ts&tests=src%2Froot_mean_square.d.ts&tests=src%2Fsample_correlation.d.ts&tests=src%2Fsample_covariance.d.ts&tests=src%2Fsample_kurtosis.d.ts&tests=src%2Fsample_skewness.d.ts&tests=src%2Fsample_standard_deviation.d.ts&tests=src%2Fsample_variance.d.ts&tests=src%2Fsample_with_replacement.d.ts&tests=src%2Fsample.d.ts&tests=src%2Fshuffle_in_place.d.ts&tests=src%2Fshuffle.d.ts&tests=src%2Fsign.d.ts&tests=src%2Fstandard_deviation.d.ts&tests=src%2Fstandard_normal_table.d.ts&tests=src%2Fsubtract_from_mean.d.ts&tests=src%2Fsum_nth_power_deviations.d.ts&tests=src%2Fsum_simple.d.ts&tests=src%2Fsum.d.ts&tests=src%2Ft_test_two_sample.d.ts&tests=src%2Ft_test.d.ts&tests=src%2Funique_count_sorted.d.ts&tests=src%2Fvariance.d.ts&tests=src%2Fz_score.d.ts)